cpp/pagerank_module/algorithm_online/pagerank.hpp

```diff
@@ -0,0 +1,154 @@
+#pragma once
+
+#include <random>
+#include <unordered_set>
+
+namespace pagerank_online_alg {
+namespace {
+class PageRankData {
+  ///
+  ///@brief Context for storing data for dynamic pagerank
+  ///
+  ///
+ public:
+  void Init() {
+    walks.clear();
+    walks_counter.clear();
+    walks_table.clear();
+
+    gen = new std::mt19937(42);
```

Comment:
Why a hardcoded seed and not use random_device?
```
std::random_device rd;
std::mt19937 g(rd())
```

Reply:
It was a testing mistake, I wanted to make sure that tests are having similar output.